src/hooks/useLocation.ts

```diff
@@ -16,7 +16,7 @@ import { useReasoningFilterMenuOption, useTimeStamp } from "@/stores/urlStore";
 
 const parseExtraParams = (extraParamsStr: string) => {
   return dJSON.parse<string, ChannelData["properties"]>(
-    extraParamsStr?.replaceAll("nan", "")
+    extraParamsStr?.replace("nan", "")
```

Comment:
I guess this is gonna only will replace the first "nan" that it find. It does not seem like a fix. 

More than that, why are we replacing "nan" to "" already on frontend side? Let's fix it on backend

Reply:
Sure

Reply:
@absolutezero13 If would be great if you could open an issue on backend, thank you. I am closing this PR, you can reopen it if you think that is wrong and anyone can reconsider! thanks for contributing